tekton_configs/test-run.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-testing
+  namespace: gabrielwen
+spec:
+  pipelineSpec:
+   tasks:
+   - name: xgboost-synthetic
+     taskRef:
+      name: pytest
+     params:
+     - name: src-dir
+       value: /mnt/testdir/src/gabrielwen
+     - name: repos
+       value: kubeflow/examples@HEAD,kubeflow/testing@HEAD,kubeflow/tf-operator@HEAD
+     - name: python-path
+       value: /mnt/testdir/src/gabrielwen/kubeflow/examples/py:/mnt/testdir/src/gabrielwen/kubeflow/testing/py:/mnt/testdir/src/gabrielwen/kubeflow/tf-operator/py
```

Comment:
Use Tekton git resources. So you should have a tekton resource for each repo. You should be able to reference the resources in setting the path e.g. something like

value: /workspace/$(inputs.resources.examples.name)/py:/workspace/$(inputs.resources.testing.name)/py